zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -54,30 +54,22 @@ angular.module('zeppelinWebApp')
 
   $scope.logout = function() {
     var logoutURL = baseUrlSrv.getRestApiBase() + '/login/logout';
-    var request = new XMLHttpRequest();
-
-    //force authcBasic (if configured) to logout by setting credentials as false:false
-    request.open('post', logoutURL, true, 'false', 'false');
-    request.onreadystatechange = function() {
-      if (request.readyState === 4) {
-        if (request.status === 401 || request.status === 405 || request.status === 500) {
-          $rootScope.userName = '';
-          $rootScope.ticket.principal = '';
-          $rootScope.ticket.ticket = '';
-          $rootScope.ticket.roles = '';
-          BootstrapDialog.show({
-            message: 'Logout Success'
-          });
-          setTimeout(function() {
-            window.location.replace('/');
-          }, 1000);
-        } else {
-          request.open('post', logoutURL, true, 'false', 'false');
-          request.send();
-        }
-      }
-    };
-    request.send();
+
+    $http.get(logoutURL).error(function() {
+      //force authcBasic (if configured) to logout
+      $http.get(logoutURL).error(function() {
```

Comment:
This seems to be the browser behaviour (have tried it on Chrome, Firefox and Safari), and in all of the browsers that I've tested requires logout to be called twice.
http://stackoverflow.com/a/233551/2749712